dist/users.js

```diff
@@ -81,4 +151,11 @@ class Users {
 
 }
 
-exports.Users = Users;
\ No newline at end of file
+exports.Users = Users;
+
+function parseUser(user) {
+  console.log(user);
+  return { ...user,
+    created_at: new Date(user.created_at)
+  };
+}
```

Comment:
New line

Reply:
this file is auto-generated, so I won't think too hard about newlines here